.github/workflows/alembic-causal-ai.yml

```
@@ -0,0 +1,173 @@
+        with:
+          name: causal-insights-${{ github.run_id }}
+          path: reports/causal_insights/
+          retention-days: 30
+      
+      - name: Send Slack Notification
+        if: ${{ failure() }}
+        uses: 8398a7/action-slack@v3
+        with:
+          status: ${{ job.status }}
+          text: 'Alembic Causal AI System encountered an issue'
+          webhook_url: ${{ secrets.SLACK_WEBHOOK_URL }}
```

actionlint (1.7.7)

input "webhook_url" is not defined in action "8398a7/action-slack@v3". available inputs are "author_name", "channel", "custom_payload", "fields", "github_base_url", "github_token", "icon_emoji", "icon_url", "if_mention", "job_name", "mention", "status", "text", "username"

(action)